### DISCUSSION TITLE
code-server support browser is?

### DISCUSSION THREAD

**Discussion by alanhe421** [coder/code-server#5798] [Q&A]
it's better to have a browser compatibility post.

**Comment by jsjoeio** [comment]
We run our e2e tests against Chrome, Firefox and Safari. Which browser are you asking about?

**Reply by alanhe421** [reply]
I would like to know the list of supported browsers and version information.
If not, will there be a hint?

According to the information you said, the browser list is Chrome, Firefox， Safari， and Edge(chromium).

**Reply by jsjoeio** [reply]
Yup, that's it as far as I know!
